src/core_ocean/shared/mpas_ocn_diagnostics.F

```diff
@@ -2146,11 +2149,11 @@ subroutine ocn_compute_land_ice_flux_input_fields(meshPool, statePool, &
          cell2 = cellsOnEdge(2, iEdge)
 
          ! top drag tau = - CD*|u|*u, where |u| = sqrt(2*KE) = sqrt(KE1 + KE2) from the neighboring cells
-         velocityMagnitude = sqrt(kineticEnergyCell(1,cell1) + kineticEnergyCell(1,cell2))
+         velocityMagnitude = sqrt(kineticEnergyCell(minLevelCell(cell1),cell1) + kineticEnergyCell(minLevelCell(cell1),cell2))
```

Comment:
Should this line be changed to:
```
velocityMagnitude = sqrt(kineticEnergyCell(minLevelCell(cell1),cell1) + kineticEnergyCell(minLevelCell(cell2),cell2))
```

In other words, should the 2nd `minLevelCell(cell1)` be changed to `minLevelCell(cell2)` to match the `cell2` in the 2nd index?

Reply:
Totally. Thanks!
```suggestion
         velocityMagnitude = sqrt(kineticEnergyCell(minLevelCell(cell1),cell1) + kineticEnergyCell(minLevelCell(cell2),cell2))
```